cypress/e2e/room_board/shareBoardCardLinkInRoom.feature

```diff
@@ -14,16 +14,16 @@ Feature: Room Board - To share a board card link
         When I click on FAB to create new room
         Then I see room creation page
         When I enter the room name '<room_name>'
-        When I click on the button to save the room
+        When I click on the button Save Room
```

Comment:
we defined: only first letter of the elements name has to be in capitals.
So here it should be "button Save room"
This has to be corrected for all "button Save room"

Reply:
done